microsoft-edge/devtools-guide-chromium/progressive-web-apps/protocol-handlers.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Test Progressive Web App (PWA) protocol handling
+description: Use the Application tool to test custom protocols you've defined in your PWA web app manifest.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 08/23/2022
+---
+
+# Test Progressive Web App (PWA) protocol handling
+
+This article assumes that you have already defined protocol handlers in your PWA web app manifest and are debugging your app with the DevTools. To learn about how to define and register protocols in your PWA see [Handle protocols in Progressive Web Apps](../../progressive-web-apps-chromium/how-to/handle-protocols.md).  
+
+Use the **Application** tool to verify and test that Microsoft Edge has successfully registered your app as a handler for custom protocols defined in your web app manifest.
```

Comment:
Is this only for **custom** protocols (i.e. `web+`), or all types of protocols? I wasn't sure if the word "custom" was needed here.

Reply:
This should work for any protocol, not just custom ones. It would be good to remove the word "custom" indeed.

Reply:
```suggestion
Use the **Application** tool to verify and test that Microsoft Edge has successfully registered your app as a handler for protocols defined in your web app manifest.
```